failed to compile targeting avr

```
error[E0425]: cannot find value `IPIO2` in this scope
   --> C:\Users\djdis\.cargo\registry\src\github.com-1ecc6299db9ec823\libm-0.2.1\src\math\rem_pio2_large.rs:264:16
    |
123 | / const PIO2: [f64; 8] = [
124 | |     1.57079625129699707031e+00, /* 0x3FF921FB, 0x40000000 */
125 | |     7.54978941586159635335e-08, /* 0x3E74442D, 0x00000000 */
126 | |     5.39030252995776476554e-15, /* 0x3CF84698, 0x80000000 */
...   |
131 | |     2.16741683877804819444e-51, /* 0x3569F31D, 0x00000000 */
132 | | ];
    | |__- similarly named constant `PIO2` defined here
...
264 |               i!(IPIO2, j as usize) as f64
    |                  ^^^^^ help: a constant with a similar name exists: `PIO2`

error[E0425]: cannot find value `IPIO2` in this scope
   --> C:\Users\djdis\.cargo\registry\src\github.com-1ecc6299db9ec823\libm-0.2.1\src\math\rem_pio2_large.rs:360:41
    |
123 | / const PIO2: [f64; 8] = [
124 | |     1.57079625129699707031e+00, /* 0x3FF921FB, 0x40000000 */
125 | |     7.54978941586159635335e-08, /* 0x3E74442D, 0x00000000 */
126 | |     5.39030252995776476554e-15, /* 0x3CF84698, 0x80000000 */
...   |
131 | |     2.16741683877804819444e-51, /* 0x3569F31D, 0x00000000 */
132 | | ];
    | |__- similarly named constant `PIO2` defined here
...
360 |                       i!(f, jx + i, =, i!(IPIO2, jv + i) as f64);
    |                                           ^^^^^ help: a constant with a similar name exists: `PIO2`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0425`.
error: could not compile `libm`
```

--- a/src/math/rem_pio2_large.rs
+++ b/src/math/rem_pio2_large.rs
@@ -27,7 +27,7 @@ const INIT_JK: [usize; 4] = [3, 4, 4, 6];
 //
 // NB: This table must have at least (e0-3)/24 + jk terms.
 //     For quad precision (e0 <= 16360, jk = 6), this is 686.
-#[cfg(target_pointer_width = "32")]
+#[cfg(any(target_pointer_width = "32", target_pointer_width = "16"))]
 const IPIO2: [i32; 66] = [
     0xA2F983, 0x6E4E44, 0x1529FC, 0x2757D1, 0xF534DD, 0xC0DB62, 0x95993C, 0x439041, 0xFE5163,
     0xABDEBB, 0xC561B7, 0x246E3A, 0x424DD2, 0xE00649, 0x2EEA09, 0xD1921C, 0xFE1DEB, 0x1CB129,

